webpack.config.js

```diff
@@ -125,11 +132,12 @@ function lorisModule(mname, entries, override=false) {
       'react': 'React',
       'react-dom': 'ReactDOM',
     },
-    node: {
-      fs: 'empty',
-    },
     devtool: 'source-map',
-    plugins: [],
+    plugins: [
+      new webpack.DefinePlugin({
+        'process.env.NODE_ENV': JSON.stringify('development'),
```

Comment:
Solves the following error in the browser:
`Webpack: Bundle.js - Uncaught ReferenceError: process is not defined`
See https://webpack.js.org/migrate/5/ and https://stackoverflow.com/questions/41359504/webpack-bundle-js-uncaught-referenceerror-process-is-not-defined